guides/common/modules/proc_migrating_project_or_smartproxy_using_backup.adoc

```diff
@@ -0,0 +1,45 @@
+[id="migrating-{project-context}-or-proxy-using_backup_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL}{nbsp}9
+
+You can migrate your existing {Project} and {SmartProxy} installation from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+
+.Terminology
+Ensure that you understand the following terms:
+
+*Source server*:: The existing {EL}{nbsp}8 system on which {Project} or {SmartProxy} is installed.
+
+*Target server*:: The new {EL}{nbsp}9 system to which {Project} or {SmartProxy} will be migrated.
+
+.Migration process overview:
+. Create a backup of the existing {Project} or {SmartProxy} on {EL}{nbsp}8.
+. Deploy a new {EL}{nbsp}9 system.
+ifdef::satellite[]
+. Restore the backup to the target server using one of these two methods:
+.. Restoring backup: You can restore both {Project} and {SmartProxy} backups with this method.
+.. Clone: You can only restore {Project} backup with this method.
+endif::[]
+ifndef::satellite[]
+. Restore the backup to the target server.
+endif::[]
+
```

Comment:
No need to diverge from the template with a subheading. This is actually what I'm missing in the procedure intro on line 4. Can you consider rewriting this to fit the introduction? I'm thinking of something like "During migration, you first create a backup [...]". If you do this, I'm sure you can also drop some of the details. Users can get an overview of the migration by looking at the procedure itself, lines 13-23 basically duplicate the steps of the procedure.